xmtp_mls/src/storage/encrypted_store/schema.rs

```diff
@@ -52,9 +52,10 @@ diesel::table! {
         installations_last_checked -> BigInt,
         added_by_inbox_id -> Text,
         welcome_id -> Nullable<BigInt>,
-        dm_inbox_id -> Nullable<Text>,
```

Comment:
I thought we exposed this out somewhere for easy look up of the the peerInboxId?

Reply:
All I really care about is this call still being performant when we list large lists of dms in RN where we have to serialize this field before sending. Can validate once we move this change up the chain.